admin/worker/run_autoscaler.go

```diff
@@ -27,18 +28,18 @@ func (w *Worker) runAutoscaler(ctx context.Context) error {
 		// if UpdatedOn is too old, the recommendation is stale and may not be trusted.
 		if time.Since(rec.UpdatedOn) >= legacyRecommendTime {
 			w.logger.Debug("skipping autoscaler: the recommendation is stale", zap.String("project_id", rec.ProjectID), zap.Time("recommendation_updated_on", rec.UpdatedOn))
-			break
+			continue
```

Comment:
Replace `break` with `continue` to allow the loop goes through all projects mentioned by the recommendation.